ui/src/app/pages/homepage/recent-workspaces.tsx

```diff
@@ -42,22 +41,30 @@ export const RecentWorkspaces = withUserProfile()
 
   render() {
     // Needs a min-height so the spinner will render when loading and position: relative so said spinner will center.
-    return <FlexRow style={{marginTop: '1rem', minHeight: 247, position: 'relative'}}>
-      {
-        !this.state.loading && this.state.recentWorkspaces.length === 0 && <NewWorkspaceButton />
-      }
-      {
-        this.state.recentWorkspaces.map(recentWorkspace => {
-          return <WorkspaceCard
-            key={recentWorkspace.workspace.name}
-            userEmail={this.props.profileState.profile.username}
-            workspace={recentWorkspace.workspace}
-            accessLevel={recentWorkspace.accessLevel}
-            reload={() => this.loadWorkspaces()}
-          />;
-        })
-      }
-      {this.state.loading && <SpinnerOverlay dark={true} />}
-    </FlexRow>;
+    return <React.Fragment>
+        {!this.state.loading && this.state.recentWorkspaces.length === 0 ? (
+          <div style={{color: colors.primary, padding: '10px 30px'}}>
+            <h2 style={{fontWeight: 600}}>Create your first Workspace</h2>
+            <div>As you create your workspaces, this area will store your most recent workspaces.</div>
+            <div>To see all workspaces created, click on <b>See all Workspaces</b> to the right.</div>
```

Comment:
The mock shows the `See all Workspaces` link greyed out, but the code leaves it enabled. *However* I think it's okay to leave it enabled. It might be worth asking Lou though.